### PR TITLE
[TestScript] Feature/20150317 phpmd copy

### PR DESCRIPTION
**Pull request by sumyapp**
PHPMDの動作確認のためのPR

### CODE REVIEW

File: app/Controller/PagesController.php [hunk 1]
```diff
@@ -74,3 +115,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
+    public function example()  {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [phpmd] The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.

File: app/Controller/PagesController.php [hunk 2]
```diff
@@ -74,3 +115,44 @@ public function display() {
 		}
 	}
 }
+
+/**
+ * CyclomaticComplexity
+ */
+// Cyclomatic Complexity = 12
+class Foo {
```
Comment (sumyapp) [inline app/Controller/PagesController.php]: [code_sniffer] Each class must be in a file by itself

Comment (sumyapp) [inline app/Controller/PagesController.php]: [code_sniffer] Each class must be in a namespace of at least one level (a top-level vendor name)

File: app/Controller/PagesController.php (hunk 1 above)
Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.

Comment (clear-nishikubo) [inline app/Controller/PagesController.php]: [development] [phpmd] The method example() has a Cyclomatic Complexity of 11. The configured cyclomatic complexity threshold is 10.

**Pull request closed without merging**